lib/matplotlib/lines.py

```diff
@@ -7,32 +7,29 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-import six
```

Comment:
The PEP8 order is stdlib (warnings), third-party (numpy, six) and internals (everything else).  Do we prefer always putting six first?  (I don't really care.)